tests/coverage/reachable/bounds/reachable_pass/test.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// This test checks that kani injects a reachability check for
+// index-out-of-bounds checks and that it reports ones that are unreachable.
```

Comment:
How do we tell whether a check such as index-out-of-bounds check is unreachable from the coverage result?

Reply:
The comments are redundant. Thanks for pointing that out, shall correct them.